addons/xeh/fnc_preStart.sqf

```diff
@@ -65,4 +65,7 @@ with uiNamespace do {
             diag_log text format ["[XEH]: %1 does not support Extended Event Handlers! Addon: %2", _classname, _addon];
         };
     } forEach (true call CBA_fnc_supportMonitor);
+
+    //compile and cache configFile eventhandlers as they won't change from here on
+    GVAR(configFileEventHandlers) = compileFinal str (configFile call CBA_fnc_compileEventHandlers);
```

Comment:
This func does several `isServer`/`isDedicated` checks to selectivly compile
`serverInit` xeh
https://github.com/CBATeam/CBA_A3/blob/master/addons/xeh/fnc_compileEventHandlers.sqf#L48-L57
At preStart everyone is a server